about.html

```diff
@@ -41,7 +41,7 @@ <h4>Become A Member</h4>
         <p>To take part in our ticketed events, you'll want to <a href="https://cssbham.com/join">click here</a> to
         purchase a membership; it's only £5 for the entire year! 😲</p>
         <p>As a member, you'll also be able to vote in our elections (held during our AGMs and EGMs) and
-            run for a position on the <a href="https://cssbham.com/committee">committee</a>. 🗳</p>
+        run for a position on the <a href="https://cssbham.com/committee">committee</a>. 🗳</p>
```

Comment:
Can you find a replacement emoji for this line. It doesn't render well for me

Reply:
Replace with 📝?

Reply:
I think it's meant to be 🗳️

Reply:
Yes, that is the correct one
But for the replacement do we want to use the memo emoji? (📝)

Reply:
Wait, do they render the same to you? It's probably just a me issue then

Reply:
The ballot renders correctly for me, but it may be worth changing as it may be an OS thing, and if so then we want the emojis to be more universal for users.